Apps/Inc/UpdateDisplay.h

```diff
@@ -0,0 +1,112 @@
+/** 
+ * @copyright Copyright (c) 2022 UT Longhorn Racing Solar
+ * @file UpdateDisplay.h
+ * @brief Function prototypes for the display application.
+ * 
+ * This contains function prototypes relevant to the UpdateDisplay
+ * application. Call assertUpdateDisplayError after calling any of the
+ * functions in this application.
+ * 
+ * @author Ishan Deshpande (IshDeshpa)
+ * @author Roie Gal (Cam0Cow)
+ * @author Nathaniel Delgado (NathanielDelgado)
+*/
+
+#ifndef __UPDATE_DISPLAY_H
+#define __UPDATE_DISPLAY_H
+
+#include "os.h"
+#include "common.h"
+#include "Tasks.h"
+
+#include "Display.h"
+#include "Contactors.h"
+
+/**
+ * Error types
+ */
+typedef enum{
+	UPDATEDISPLAY_ERR_NONE			= 0,
+	UPDATEDISPLAY_ERR_FIFO_PUT		=-1,	// Error putting command in fifo
+	UPDATEDISPLAY_ERR_FIFO_POP		=-2,	// Error popping command from fifo
+	UPDATEDISPLAY_ERR_PARSE_COMP	=-3,	// Error parsing component/val in SetComponent
+} UpdateDisplayError_e;
+
+/**
+ * @brief Initializes UpdateDisplay application
+ * @returns UpdateDisplayError_e
+ */
+
+UpdateDisplayError_e UpdateDisplay_Init();
+
+/**
+ * @brief Selects visible page on the display
+ * @param page which page to select
+ * @returns UpdateDisplayError_e
+ */
+UpdateDisplayError_e UpdateDisplay_SetPage(Page_e page);
+
+/**
+ * @brief Sets the state of charge value on the display
+ * @param percent charge as a percent
+ * @returns UpdateDisplayError_e
+ */
+UpdateDisplayError_e UpdateDisplay_SetSOC(uint8_t percent);
```

Comment:
Indicate degree of precision in the header comment here. ie, if we have 0 digits of precision (100 = 100.0%), this should be indicated.